src/ui/header/header.tsx

```diff
@@ -1,18 +1,29 @@
 import { useState } from 'react';
```

Comment:
better to move Header to Features due to some navigation logic not specific for UI components